examples/ensemble/plot_feature_transformation.py

```diff
@@ -0,0 +1,103 @@
+"""Use trees to transform your features
```

Comment:
You need to have a title like in the other plot examples which will be used in the example gallery.